content/posts/newsletter/2018-11-30.md

```diff
@@ -0,0 +1,102 @@
+---
+title: The XMPP Newsletter, 30 November 2018
+date: 2018-11-30 00:00
+author: jcbrand
+blog_id: newsletter 
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter.
+
+If you have an article, tutorial or blog post you'd like us to include in the
+newsletter, please [submit it on the XMPP wiki](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## News
+
+Paul Schaub has written about a [QR-code generator for OMEMO](https://blog.jabberhead.tk/2018/11/03/qr-code-generator-for-omemo/)
+which he has been working on. It encodes the Jabber-ID of the owner as well as any OMEMO
+fingerprints they choose to include.
+
+JC Brand has written about the [recent XMPP sprint held in Dusseldorf](https://opkode.com/blog/xmpp-sprint-dusseldorf/).
+Featuring sushi, ramen, whiskey, protocol discussions, translations, bugfixes, documentation, releases, hacks and coding.
+
+The [Monal Android and Mac client now shows a simple introduction to
+XMPP](https://monal.im/blog/the-welcome-screen/) with
+lovely illustrations of Peter rabbit, as he enters and escapes Mr McGregor's garden.
+There are also new [empty state screens](https://monal.im/blog/the-empty-state/) with similar illustrations.
+
+In the Netherlands, a chat service called IronChat, with an Android client which
+is apparently a fork of [Conversations](https://conversations.im/), was 
+compromised when the police took control of the centralized server being used
+and started a man-in-the-middle attack on the conversations. It seems the
+service was very popular with criminals, which is why the police got involved,
+and various arrests were made. [The original article in Dutch](https://nos.nl/artikel/2258309-beveiliging-door-politie-gekraakte-cryptofoons-was-twijfelachtig.html)
+and here's a [Google machine-translated version](https://translate.google.com/translate?hl=&sl=nl&tl=en&u=https%3A%2F%2Fnos.nl%2Fartikel%2F2258309-beveiliging-door-politie-gekraakte-cryptofoons-was-twijfelachtig.html).
+
+The well-known Kuketz blog in Germany wrote an article [XMPP is not the savior - but a solution](https://translate.google.com/translate?hl=&sl=auto&tl=en&u=https%3A%2F%2Fwww.kuketz-blog.de%2Fmessenger-xmpp-ist-nicht-der-heilsbringer-aber-eine-loesung%2F)
+(Google machine-translated version). Here's the [original German version](https://www.kuketz-blog.de/messenger-xmpp-ist-nicht-der-heilsbringer-aber-eine-loesung/).
+It's in part a response to [XMPP: Admin-in-the-middle](https://infosec-handbook.eu/blog/xmpp-aitm/), written by the InfoSec Handbook
+which details how XMPP server administrators have access to user's metadata and
+other sensitive information.
+
+The German email-provider mailbox.org, has [unveiled a new revamped website](https://mailbox.org/en/post/a-new-logo-a-new-website-new-software-versions-and-new-features-at-mailbox-org)
```

Comment:
"email-provider" or "email provider"? Not sure but I think the dash should not be there.

Reply:
Thanks, fixed.